packages/@okta/vuepress-theme-prose/global-components/Login.vue

```diff
@@ -47,6 +47,7 @@
 
 <script>
 export default {
+  name: 'Login',
```

Comment:
nit: since we seem to be using double quotes in this repo:
```suggestion
  name: "Login",
```